src/seedu/addressbook/commands/InfoCommand.java

```diff
@@ -0,0 +1,15 @@
+package seedu.addressbook.commands;
+
+public class InfoCommand extends Command{
+    public static final String COMMAND_WORD= "info";
+    public static final String MESSAGE_USAGE= COMMAND_WORD+": print out several information"+"\n"+"Example: "+COMMAND_WORD ;
+    public static final String INFO ="This is the addressbook level2 version 0.1 "
```

Comment:
I would suggest not to hardcode the version in the INFO string as it is supposed to change with the growth of an application. I would suggest to keep it in a separate variable and dynamically generating the INFO string with formatting.